src/containers/organizations/components/CreateOrganizationComponent.js

```diff
@@ -95,12 +107,61 @@ class CreateOrganization extends React.Component {
 
   onClickCreate = () => {
```

Comment:
Look at `CatalogComponent` `props.onSubmitSearch` for a comparable example. `onSubmitSearch` dispatches a `CATALOG_SEARCH_REQUEST`. The router is triggered first, and it sets the async state to `LOADING` - see `CatalogReducer` and `CatalogComponent.render` to see how the reducer modifies the `AsyncContent` component.